docs/tutorials/elasticdl_local.md

```diff
@@ -54,66 +60,101 @@ elasticdl train \
-  --worker_resource_limit="cpu=1,memory=3072Mi" \
+  --worker_resource_request="cpu=0.4,memory=1024Mi" \
+  --worker_resource_limit="cpu=1,memory=2048Mi" \
+  --ps_resource_request="cpu=0.2,memory=1024Mi" \
+  --ps_resource_limit="cpu=1,memory=2048Mi" \
   --minibatch_size=64 \
   --num_minibatches_per_task=2 \
-  --num_workers=2 \
-  --checkpoint_steps=10 \
-  --evaluation_steps=15 \
-  --grads_to_wait=2 \
+  --num_ps_pods=1 \
+  --num_workers=1 \
+  --evaluation_steps=50 \
+  --grads_to_wait=1 \
   --job_name=test-mnist \
   --log_level=INFO \
   --image_pull_policy=Never \
-  --output=model_output
+  --distribution_strategy=ParameterServerStrategy
 ```
 
-We can also use the pre-built docker image to run the ElasticDL job.
-In the following command, the pre-built image name is `elasticdl:mnist`, the absolute model zoo path inside the docker is `/model_zoo`.
+`image_base` is the base docke image argument. A new image will be built based on it each time while submitting the Elastic job.
 
-```bash
-elasticdl train \
-  --image_name=elasticdl:mnist \
-  --docker_base_url=${DOCKER_BASE_URL} \
-  --docker_tlscert=${DOCKER_TLSCERT} \
-  --docker_tlskey=${DOCKER_TLSKEY} \
-  --model_zoo=/model_zoo \
-  --model_def=mnist_functional_api.mnist_functional_api.custom_model \
-  --training_data=/data/mnist/train \
-  --validation_data=/data/mnist/test \
-  --num_epochs=2 \
-  --master_resource_request="cpu=400m,memory=1024Mi" \
-  --master_resource_limit="cpu=1,memory=2048Mi" \
-  --worker_resource_request="cpu=400m,memory=2048Mi" \
-  --worker_resource_limit="cpu=1,memory=3072Mi" \
-  --minibatch_size=64 \
-  --num_minibatches_per_task=2 \
-  --num_workers=2 \
-  --checkpoint_steps=10 \
-  --evaluation_steps=15 \
-  --grads_to_wait=2 \
-  --job_name=test-mnist \
-  --log_level=INFO \
-  --image_pull_policy=Never \
-  --output=model_output
-```
+We use the model predefined in model zoo directory. The model definition will be packed into the new docker image. The training and validation data are packaged to the base docker image already. We could use them directly.
+
+In this example, we use parameter server strategy. We launch a master pod, a parameter server(PS) pod and a worker pod. The worker pod gets model parameters from the PS pod, computes gradients and sends computed gradients to the PS pod. The PS pod iteratively updates these model parameters using gradients sent by the worker pod. For more details about parameter server strategy, please refer to the [design doc](https://github.com/sql-machine-learning/elasticdl/blob/develop/docs/designs/parameter_server.md).
 
 ### Check job status
 
-After submitting the job to Minikube, you can run following command to check the status of each pod:
+After submitting the job to Minikube, we can run following command to check the status of each pod:
 
 ```bash
 kubectl get pods
 ```
 
-You should see information on each pod like the following:
+We will see information on each pod like the following:
```

Comment:
We will see the information on each pod such as:

Reply:
Done